### ISSUE TITLE
ctags / taglist integration

### ISSUE THREAD

**Issue by devth** [topfunky/PeepOpen-Issues#177]
Using PeepOpen's awesome fuzzy matching on ctags would be amazing and irresistible!

**Comment by tlamadon** [comment]
I agree to that! If PeepOpen could also parse an extra file (ctags or else) it would great. One could then write a script that populates the list. I guess one can do that from the sources...
